mountpoint-s3/tests/fuse_tests/write_test.rs

```diff
@@ -435,6 +437,65 @@ where
     assert_eq!(err.raw_os_error(), Some(libc::ENOENT));
 }
 
+fn write_with_storage_class_test<F>(creator_fn: F, storage_class: &str)
+where
+    F: FnOnce(&str, TestSessionConfig) -> (TempDir, BackgroundSession, TestClientBox),
+{
+    const KEY: &str = "new.txt";
+
+    let filesystem_config: S3FilesystemConfig = S3FilesystemConfig {
```

Comment:
nit: I believe the type annotation (`: S3FilesystemConfig`) is redundant here.

Reply:
super nit: this is only setting one field, I would probably inline it in `TestSessionConfig` (line 452)